From ben-linux@fluff.org  Mon Nov 26 23:05:09 2007
From: Ben Dooks <ben-linux@fluff.org>
Date: Mon, 19 Nov 2007 22:28:13 +0000
Subject: USB: s3c2410 gadget: Header move fixups
To: Linux USB Devel <linux-usb-devel@lists.sourceforge.net>
Cc: vince@simtec.co.uk, Greg KH <greg@kroah.com>, Ben Dooks <ben-linux@fluff.org>
Message-ID: <20071119222846.887780944@fluff.org>


Fixup the fallout from the arch moves earlier in the kernel
series.

Signed-off-by: Ben Dooks <ben-linux@fluff.org>
Acked-by: David Brownell <dbrownell@users.sourceforge.net>

---
 drivers/usb/gadget/s3c2410_udc.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/drivers/usb/gadget/s3c2410_udc.c
+++ b/drivers/usb/gadget/s3c2410_udc.c
@@ -52,10 +52,10 @@
 #include <asm/arch/irqs.h>
 
 #include <asm/arch/hardware.h>
-#include <asm/arch/regs-clock.h>
 #include <asm/arch/regs-gpio.h>
-#include <asm/arch/regs-udc.h>
-#include <asm/arch/udc.h>
+
+#include <asm/plat-s3c24xx/regs-udc.h>
+#include <asm/plat-s3c24xx/udc.h>
 
 #include <asm/mach-types.h>
 
